plugins/pulp_docker/plugins/importers/upload.py

```diff
@@ -52,6 +52,21 @@ def get_models(metadata, mask_id=''):
     return images
 
 
+def graceful_extract_from_archive(archive, path):
+    possible_paths = [
```

Comment:
since the ``path`` will always have the form: ``<image_id>/<filename>``, I don't think any of these three choices would cover the possibility that the path in the archive starts with ``./``.